src/taipy/core/cycle/_cycle_manager.py

```diff
@@ -57,7 +54,7 @@ def _get_all_by(cls, by, version_number: Optional[str] = "all") -> List[Cycle]:
 
     @classmethod
     def _get_or_create(
-        cls, frequency: Frequency, creation_date: Optional[datetime] = None, display_name: Optional[str] = None
+        cls, frequency: Frequency, creation_date: Optional[datetime] = None, name: Optional[str] = None
```

Comment:
Are we using the `name` property, or exposing it in the documentation in any way?

I know it's different from `label`, but I don't see the purpose of the `name` (yet).

Reply:
The name is an optional attribute of some entities. It s for an end user to name the entities he/she creates so he can differentiate them and access them. He/she does not want to use an id for that.
It is documented in the reference manual.

However, the "display name" was never used or documented. So I removed it.